lists/vn/vn-most.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": {
+    "en": "Ministry of Science and Technology",
+    "local": " Bộ Khoa học và Công nghệ"
+  },
+  "url": "https://www.most.gov.vn/en/Pages/home.aspx",
+  "description": {
+    "en": "The Ministry of Science and Technology (MOST, Vietnamese: Bộ Khoa học và Công nghệ) is a government ministry in Vietnam responsible for state administration of science and technology activities; development of science and technology potentials; intellectual property; standards, metrology and quality control; atomic energy, radiation and nuclear safety"
+  },
+  "coverage": [
+    "VN"
+  ],
+  "code": "VN-MOST",
+  "listType": "primary",
```

Comment:
Please change to 'secondary'. (For info: https://org-idguide-handbook.readthedocs.io/en/latest/metadata/#list-type)

Reply:
Edited in latest commit.